### DISCUSSION TITLE
handle Separate Spaces/Every Desktop separate

### DISCUSSION THREAD

**Discussion by mleenings** [dwarvesf/hidden#221] [Ideas]
My Idea: An Option, that show/hide will work only for a special desktop (if you have multi desktops) or maybe with the misson control > separate spaces... or work with all together (all together is now).
Use Case when i cast my display, i want only hide the menu bar of that display which i cast, so that the other not distracted from mine menu bar.

**Comment by ghost** [comment]
I think this app is no longer getting much love and attention from its developers 😞 

@viettrungphan are you still planning to make updates / improvements to this app?
